2025-03-21 15:44:32,583 - root - INFO - Logging has been configured.
2025-03-21 15:44:32,613 - __main__ - INFO - Loaded config.json - Starting execution with id d316b280-41cd-4f83-9ee3-521d1654086d
2025-03-21 15:44:32,614 - configuration_manager - INFO - Configuration: 
2025-03-21 15:44:32,615 - configuration_manager - INFO - * Runtime ASM model: SafetyEnforcerModel.asm
2025-03-21 15:44:32,616 - model_uploader - INFO - Enforcer initialized with domain: http://localhost:8080/
2025-03-21 15:44:32,616 - model_uploader - INFO - Runtime model: SafetyEnforcerModel.asm
2025-03-21 15:44:32,617 - model_uploader - INFO - 
2025-03-21 15:44:32,681 - model_uploader - INFO - Uploaded file: ../../resources/models\SafetyEnforcerModel.asm
2025-03-21 15:44:32,800 - model_uploader - INFO - Uploaded file: ../../resources/models\..\libraries\StandardLibrary.asm
2025-03-21 15:44:32,801 - __main__ - INFO - --Starting new test run--
2025-03-21 15:44:32,899 - enforcer - INFO - Execution started with ID: 4
2025-03-21 15:44:32,899 - __main__ - INFO - --Executing new step--
2025-03-21 15:44:46,187 - enforcer - INFO - ASM step performed for ID 4 with delay 127.70 ms
2025-03-21 15:44:46,188 - enforcer - ERROR - ASM step execution failed: 'outAction'
2025-03-21 15:44:46,188 - __main__ - ERROR - Failed to connect to the server - Executing the test runs WITHOUT the model
2025-03-21 15:44:46,189 - __main__ - INFO - --Starting new test run--
2025-03-21 15:44:46,190 - __main__ - INFO - --Executing new step--
